topcoffea/cfg/data_samples.cfg

```diff
@@ -88,23 +88,23 @@ root://ndcms.crc.nd.edu/
 
 # 2018 data
 # See: https://twiki.cern.ch/twiki/bin/view/CMS/PdmVDatasetsUL2018
-root://ndcms.crc.nd.edu/
-../../topcoffea/json/data_samples/2018/DoubleMuon_A-UL2018.json
-../../topcoffea/json/data_samples/2018/DoubleMuon_B-UL2018.json
-../../topcoffea/json/data_samples/2018/DoubleMuon_C-UL2018.json
-../../topcoffea/json/data_samples/2018/DoubleMuon_D-UL2018.json
-
-../../topcoffea/json/data_samples/2018/EGamma_A-UL2018.json
-../../topcoffea/json/data_samples/2018/EGamma_B-UL2018.json
-../../topcoffea/json/data_samples/2018/EGamma_C-UL2018.json
-../../topcoffea/json/data_samples/2018/EGamma_D-UL2018.json
-
-../../topcoffea/json/data_samples/2018/MuonEG_A-UL2018.json
-../../topcoffea/json/data_samples/2018/MuonEG_B-UL2018.json
-../../topcoffea/json/data_samples/2018/MuonEG_C-UL2018.json
-../../topcoffea/json/data_samples/2018/MuonEG_D-UL2018.json
-
-../../topcoffea/json/data_samples/2018/SingleMuon_A-UL2018.json
-../../topcoffea/json/data_samples/2018/SingleMuon_B-UL2018.json
-../../topcoffea/json/data_samples/2018/SingleMuon_C-UL2018.json
-../../topcoffea/json/data_samples/2018/SingleMuon_D-UL2018.json
+#root://ndcms.crc.nd.edu/
+#../../topcoffea/json/data_samples/2018/DoubleMuon_A-UL2018.json
+#../../topcoffea/json/data_samples/2018/DoubleMuon_B-UL2018.json
+#../../topcoffea/json/data_samples/2018/DoubleMuon_C-UL2018.json
+#../../topcoffea/json/data_samples/2018/DoubleMuon_D-UL2018.json
+#
+#../../topcoffea/json/data_samples/2018/EGamma_A-UL2018.json
+#../../topcoffea/json/data_samples/2018/EGamma_B-UL2018.json
+#../../topcoffea/json/data_samples/2018/EGamma_C-UL2018.json
+#../../topcoffea/json/data_samples/2018/EGamma_D-UL2018.json
+#
+#../../topcoffea/json/data_samples/2018/MuonEG_A-UL2018.json
+#../../topcoffea/json/data_samples/2018/MuonEG_B-UL2018.json
+#../../topcoffea/json/data_samples/2018/MuonEG_C-UL2018.json
+#../../topcoffea/json/data_samples/2018/MuonEG_D-UL2018.json
+#
+#../../topcoffea/json/data_samples/2018/SingleMuon_A-UL2018.json
+#../../topcoffea/json/data_samples/2018/SingleMuon_B-UL2018.json
+#../../topcoffea/json/data_samples/2018/SingleMuon_C-UL2018.json
+#../../topcoffea/json/data_samples/2018/SingleMuon_D-UL2018.json
```

Comment:
Just wondering why these JSONs are now commented by default? Was that intentional?

Reply:
fixed with 80bbed74e99fe51bc931a15c9f7ddaec83a66c67